MvvmCross/Core/Binding/Bindings/Target/MvxWithEventPropertyInfoTargetBinding.cs

```diff
@@ -53,26 +71,64 @@ public override void SubscribeToEvents()
                 return;
 
             var viewType = target.GetType();
-            var eventName = this.TargetPropertyInfo.Name + "Changed";
+
+            // Try a specific [PropertyName]Changed event first.
+            var eventInfo = GetNamedPropertyChangedEvent(viewType, this.TargetPropertyInfo.Name);
+            if (eventInfo != null)
+                this._subscription = eventInfo.WeakSubscribe(target, OnValueChanged);
+            else if (target is INotifyPropertyChanged notifyPropertyChangedTarget)
+            {
+                // If target implements INPC then try a generic PropertyChanged event next.
+                eventInfo = GetPropertyChangedEvent(viewType);
```

Comment:
Checking for the actual event and constructing `EventInfo` is not strictly necessary in this case since the subscription only needs the target and we have already checked for the `INotifyPropertyChanged` interface, but it seemed more consistent to do it this way. Let me know if you prefer to have it changed.